internal/prompt/prompt.go

```diff
@@ -17,23 +18,34 @@ type Prompter interface {
 	Select(title, message string, choices []string, defaultResponse string) (string, *failures.Failure)
 	Confirm(title, message string, defaultChoice bool) (bool, *failures.Failure)
 	InputSecret(title, message string, flags ...ValidatorFlag) (string, *failures.Failure)
+	IsInteractive() bool
 }
 
-// FailPromptUnknownValidator handles unknown validator erros
+// FailPromptUnknownValidator handles unknown validator errors
 var FailPromptUnknownValidator = failures.Type("prompt.unknownvalidator")
 
+// FailPromptNonInteractive handles errors during nonInteractive mode
+var FailPromptNonInteractive = failures.Type("prompt.unknown.nonInteractive")
```

Comment:
Please use errors. Are you trying to make my conflict resolution even more painful? :P

Reply:
:)  No, just tried to stay consistent to this function, but I guess it's simple enough to transform everything to errors.

Reply:
Actually, it is not quite soooo simple to transform everything to errors here.  Then I would have to change all the `Prompter` functions' signatures to return error instead of `*failure.Failure`.  And that is a huge rabbit hole, and work that you'll have already done in your PR.  I think we should either merge yours first, and then I can fix it, or just be very mindful if we do it the other way round...